packages/persist/lib/server.ts

```diff
@@ -21,6 +21,8 @@ const maxSizeJsonRecords = '100mb';
 
 export const server = express();
 
+server.set('query parser', 'extended');
```

Comment:
is this necessary for the other apps?

Reply:
I don't think so, we mostly use body for jobs and runner